cellpack/autopack/Environment.py

```diff
@@ -143,7 +143,11 @@ def __init__(self, config=None, recipe=None):
         self.name = name
         self.version = recipe.get("version", "default")
         # saving/pickle option
-        self.saveResult = "out" in config
+        self.saveResult = (
+            "out" in config
+            and not config["save_analyze_result"]
```

Comment:
Why do we have these additional conditions here? Wouldn't this prevent saving simularium files completely if there is more than one packing?

Reply:
it's because the saving happens from the Analysis class instead of the Env class